presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java

```diff
@@ -190,6 +198,50 @@ public void testDuplicatedRowCreateTable()
                 "line 1:49: Column name 'orderkey' specified more than once");
     }
 
+    @Test
+    public void testCreateTableAndShowWithNotNullColumn()
```

Comment:
We don't want to add table creation tests to smoke tests. Instead, table creation should go in connector specific tests or `AbstractTestDistributedQueries`.